Assets/HoloToolkit/Input/Tests/Scripts/ChangeColorOnSelect.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+using HoloToolkit.Unity;
+
+/// <summary>
+/// Controls object appearance by changing its color when selected
+/// and returning it to its original color when a different object is selected
+/// </summary>
+public class ChangeColorOnSelect : Interactable
+{
+    [Tooltip( "Object color changes to this when focused." )]
+    [SerializeField]
+    private Color selectedColor = Color.red;
+
+    private Material materialInstance;
+
+    private Color originalColor;
+
+    private void Start()
+    {
+        materialInstance = GetComponent<Renderer>().material;
```

Comment:
Again, this class was renamed, and probably should be addressing in a separate PR or Issue in the main branch of the HTK.